### PR TITLE
Updated extension folder

### PR DESCRIPTION
**Pull request by Bbrighi2**
Changes made to extension folder

### CODE REVIEW

File: extension/contentScript.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+if (!messageListener) {
+  window.addEventListener(
+    'message',
+    (messageEvent) => {
+      if (messageEvent.source === window) {
+        chrome.runtime.sendMessage(messageEvent.data);
+      }
+    },
+    false
```
Comment (myeon7) [inline extension/contentScript.js:11]: What does the 'false' do here?

**Review by THLyon — APPROVED** [review]

File: extension/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+} from 'solid-js';
+import { Header } from './components/Header';
+import { Inspect } from './components/Inspect';
+import { Graph } from './components/Graph';
+import { Navbar } from './components/Navbar';
+import { SolidComponent, TabType, OrientType } from './types';
+import './styles/main.scss';
+
+export const SolidStructure: SolidComponent = () => {
+  // const [children, root] = createRoot(() => [props.children, getOwner()!]);
```
Comment (myeon7) [inline extension/src/App.tsx:17]: Are we accessing browser-rendered signals through createRoot()?

**Review by myeon7 — COMMENTED** [review]
1) (extension/contentScript.js) Functionality of having 'false' as the second argument of the window.addEventListener()
2) (extension/src/App.tsx) Is createRoot() used to access the signals from browser-rendered application?

File: extension/contentScript.js (hunk 1 above)
Comment (Bbrighi2) [reply extension/contentScript.js:11]: Per MDN: A boolean value indicating that the listener should be invoked at most once after being added.  It is optional and the default value is set to false so I will remove it.

File: extension/src/App.tsx (hunk 2 above)
Comment (Bbrighi2) [reply extension/src/App.tsx:17]: The idea is that we are wrapping the target application in our SolidStructure component and with the addition of a createRoot, we will have access to all Signals and Signals of children components.

**Review by myeon7 — APPROVED** [review]
All checked. Great Work!

**Pull request merged**